src/sidebar/host-config.js

```diff
@@ -10,6 +10,9 @@ function hostPageConfig(window) {
   const configJSON = queryString.parse(window.location.search).config;
   const config = JSON.parse(configJSON || '{}');
 
+  // temp: hack to force user
+  config.focusedUser = 'kyle';
```

Comment:
So this hack obviously can't stay, but I would like to know the right way to ad-hoc a setting in a config when running locally. I was confused that this didn't work when placing it in the script tag in `live-reload-server.js`